### PR TITLE
V1.10.1

### PR DESCRIPTION
**Pull request by cole8888**
- Removed nebula and curiosity stream edge case.
- Updated highlight, preview, outro, bubbles.
- Rearanged the bubbles to make more efficient use of space.
- Rearanged intro/outro leaf nodes to make the simpler path go to outro, since outro is the most likely answer.
- Changed text in node 2.
- Changed text in node prior to preview leaf node.
- Moved and changed top text.
- Added potentially valuable information info bubble
- Added info bubbles to yaml.
- Increased text size of leaf nodes and start bubble.
- SVG file no longer renders as a mess.

### CODE REVIEW

File: SB_Flowchart.yaml [hunk 1]
```diff
@@ -2,9 +2,22 @@
 
 # titles, descriptions and text might be markdown
 
-results:  # sorted by numerical ID in orig chart
+info_bubbles:
```
Comment (cole8888) [inline SB_Flowchart.yaml:5]: No idea if I did this right.

**Review by mchangrh — COMMENTED** [review]
Is this necessary? It seems a bit convoluted with the additional clause of no on screen content

**Review by mchangrh — APPROVED** [review]
LGTM, I remember the on-screen disclaimer was added b/c users kept on submitting it.

**Comment by cole8888** [comment]
> Is this necessary? It seems a bit convoluted with the additional clause of no on screen content

Discussed on discord. Ignoring because an on-screen only sponsor can happen when no content playing, so we need to clarify it's ok to submit a sponsor in that case.

**Pull request merged**